app/code/community/Bolt/Boltpay/BoltGlobalTrait.php

```diff
@@ -23,13 +23,12 @@
 trait Bolt_Boltpay_BoltGlobalTrait {
 
     /**
-     * Returns the main helper class used by Bolt or if from a class that natively supports this function
-     * with a parameter, that helper
+     * Returns the main helper class used by Bolt
      *
      * @return Bolt_Boltpay_Helper_Data|Mage_Core_Helper_Abstract
      */
     public function boltHelper() {
         return Mage::helper('boltpay');
     }
-
+    
```

Comment:
Why new line?

Reply:
@adentong 
To be clear from a PHP functional perspective, there is no reason other than the IDE adding this.  It changes the code functionality in no way, triggers nothing in the official Magento codesniffing or lint  that we have set up, and it is not something that is easily human perceptible until looking at the diff after the commit.
 
In general, we have often seen and will continue to see many inconsequential white space edits done in several peoples commits.  I would prefer that we not to focus on this because if the counter question were asked, _What meaningful difference does it make?_ , I believe the rational answer is none.

**However, it's always ok to raise the bar on code delivery expectations across the board,** and if not adding a custom rule to our codesniffing, or adding an automated linting commit-hook that automatically detects and removes extra white-space, we can make sure moving forward to take a little extra time to edit commits to remove extra white-space, including now!

Reply:
Yes I agree that we shouldn't focus on whitespace changes too much, but this just stood out in a very small PR. My philosophy is to not change something unless there is a reason to, which for this change there is obviously not, that's why I pointed it out.

Reply:
@adentong 
Absolutely!